cypress/e2e/awx/resources/projects/project-edit-copy-sync.cy.ts

```diff
@@ -31,7 +31,7 @@ describe('Project Edit, Copy, Sync', () => {
       cy.get(`[data-cy="row-id-${project.id}"]`).within(() => {
         cy.get('[data-cy="edit-project"]').click();
       });
-      cy.verifyPageTitle('Edit Project');
+      cy.verifyPageTitle('Edit Demo Project @ 10:44:51');
```

Comment:
@tiyiprh This name is not correct. As names in tests are generated it cannot be hardcoded like here. Instead something like 
```
cy.verifyPageTitle(`Edit ${project.name}`);
```